core/container/externalbuilder/externalbuilder.go

```diff
@@ -418,10 +419,5 @@ func appendDefaultPropagateEnvironment(propagateEnvironment []string) []string {
 }
 
 func contains(propagateEnvironment []string, key string) bool {
-	for _, variable := range propagateEnvironment {
-		if key == variable {
-			return true
-		}
-	}
-	return false
+	return slices.Contains(propagateEnvironment, key)
 }
```

Comment:
it is necessary not to change the body of the contains function, but the calls themselves. the contains function is no longer needed.